pkg/api/api.go

```diff
@@ -223,6 +223,14 @@ func (hs *HttpServer) registerRoutes() {
 		// Dashboard
 		r.Group("/dashboards", func() {
 			r.Combo("/db/:slug").Get(GetDashboard).Delete(DeleteDashboard)
+
+			r.Get("/db/:dashboardId/versions", wrap(GetDashboardVersions))
```

Comment:
had to change this to `/id/:dashboardId/` not super elegant but it's better than having a slug and and id in the same URL segment,